### PR TITLE
Add colon-style emoji support

### PR DESCRIPTION
**Pull request by ddbruce**
This allows `:emoji:`s to be placed in the text of chores and notes. Colon-style emojis are often used with Slack, and so long as the emoji is standard (i.e. not a custom-added emoji to a Slack workspace), it should be converted to Unicode and show up as appropriate.

### CODE REVIEW

File: public/js/admin.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
+/* global EmojiConvertor */
+
 async function getNotes() {
+    const emoji = new EmojiConvertor();
```
Comment (justetz) [inline public/js/admin.js]: Does this not need `emoji.init_env();` as used in `updateChores` of `whiteboard.js`?

File: public/js/whiteboard.js [hunk 2]
```diff
@@ -22,12 +22,18 @@ function updateCrew(crewResponse) {
 }
 
 function updateNotes(noteResponse) {
+    let emoji = new EmojiConvertor();
+    emoji.init_env();
+    emoji.replace_mode = 'unified';
+    emoji.allow_native = true;
```
Comment (justetz) [inline public/js/whiteboard.js]: Please make a helper function with the `EmojiConvertor` constructor call and the three settings for the `emoji` object as the same setup is used in multiple places.

File: public/js/whiteboard.js [hunk 3]
```diff
@@ -42,6 +48,10 @@ function updateDate() {
 }
 
 function updateChores(choreList) {
+    let emoji = new EmojiConvertor();
+    emoji.init_env();
+    emoji.replace_mode = 'unified';
+    emoji.allow_native = true;
```
Comment (justetz) [inline public/js/whiteboard.js]: See previous comment about helper function

File: public/js/admin.js (hunk 1 above)
Comment (ddbruce) [reply public/js/admin.js]: This was tested and works. I'm not sure that it needs it due to that.

**Comment by ddbruce** [comment]
Consider switching to PNG or otherwise image emojis rather than Unicode ones

**Comment by ddbruce** [comment]
Can be reopened with fresh code if the feature is still desired.

**Pull request closed without merging**